### PR TITLE
Us124 apply prototype styling - post PO discussion

### PR DESCRIPTION
**Pull request by danielmarker**

### CODE REVIEW

File: application/static/landregistry/assets/stylesheets/landregistry.scss [hunk 1]
```diff
@@ -176,3 +176,7 @@ dl {
   padding-top: 15px;
   border-top: 1px solid #dee0e2;
 }
+
+.bolden-text{
```
Comment (endamccormack) [inline application/static/landregistry/assets/stylesheets/landregistry.scss]: bold please

File: application/templates/viewdeed.html [hunk 2]
```diff
@@ -34,14 +29,10 @@ <h3 class="heading-medium">Lender</h3>
...
+        <div class="bolden-text">
             {% for borrower in deed_data.deed.borrowers %}
-            <div class="column-third">
-              <p>
-                <b>{{ borrower.forename}} {{ borrower.middle_name}}{% if borrower.middle_name %} {% endif %}{{
-                borrower.surname}}</b><br>
-              </p>
-            </div>
+                {{ borrower.forename}} {{ borrower.middle_name}}{% if borrower.middle_name %} {% endif %}{{
+                borrower.surname}}<br>
```
Comment (endamccormack) [inline application/templates/viewdeed.html]: self closing tags would always include a close i.e. `<br/>`

Also strictly I don't think you should use br tags. They are frowned upon

Comment (endamccormack) [inline application/templates/viewdeed.html]: better including the entire thing in a `<p>`, this should give it natural seperation

**Comment by endamccormack** [comment]
:+1:

File: application/templates/viewdeed.html [hunk 3]
```diff
@@ -78,7 +67,7 @@ <h3 class="heading-small">Additional provisions:</h3>
                 <p class="bordered-output-not-present" id="registrarsig">[Will be applied when the charge takes effect]</p>
             </div>
             <div class="text">
-              <label>E-MD ref: {{deed_data.deed.md_ref}}</label><br>
+              <label>{{deed_data.deed.md_ref}}</label><br>
```
Comment (endamccormack) [inline application/templates/viewdeed.html]: no `<br/>` please

**Comment by danielmarker** [comment]
Merging post PO Sign Off Talk.

**Pull request merged**